ops/localenv/main.go

```diff
@@ -30,7 +30,7 @@ func main() {
 	switch strings.ToLower(os.Args[1]) {
 	// create k8s cluster + resources
 	case "create":
-		run("create registry", "k3d", "registry", "create", "registry.localhost", "--port", "12345")
+		run("create registry", "k3d", "registry", "create", "registry.localhost", "--port", "12345", "--default-network", "podman")
```

Comment:
TODO: need to revert that/make it detect podman

Reply:
What do you want to do here?